File: years/2017/elixir/test/aoc/day_10_test.exs
defmodule AoC.Day10.Test do
  @moduledoc false

  use ExUnit.Case, async: false

  # comment this out to always log to the console
  @moduletag :capture_log

  describe "part 1" do
    test "with sample data" do
      assert AoC.Day10.part_1(["../data/10.txt"]) == 1
    end
  end

  describe "part 2" do
    test "with sample data" do
      assert AoC.Day10.part_2(["../data/10.txt"]) == 1
    end
  end
end

File: lang/clojure/skeleton/test/day_10_test.clj
(ns day-10-test
  (:require
   [clojure.test :refer [deftest is testing]]
   [day-10 :refer [part1 part2]]))

(deftest day-10-test
  (testing "sample data"
    (is (= 0 (part1 "../data/10-sample.txt")))
    (is (= 0 (part2 "../data/10-sample.txt")))))
